tests/data/serialization/ndjson/test_text.py

```diff
@@ -13,40 +13,92 @@ def test_serialization():
                       text="This is a test",
                   ),
                   annotations=[
-                      ClassificationAnnotation(
-                          name="radio_question_geo",
-                          confidence=0.5,
-                          value=Text(answer="first_radio_answer"))
+                      ClassificationAnnotation(name="free_text_annotation",
```

Comment:
Is this the best place to put this test? `test_text.py` is testing for Text DataRow cases, so it doesn't seem like a good place to put tests on text _classification_.